package.json

```diff
@@ -121,13 +121,13 @@
     "**/*.ts": [
       "tsc",
       "tslint --fix -p . -c tslint.json",
-      "test-unit",
+      "npm run test-unit",
```

Comment:
What are these changes for? Did they fix the commit-hook for you?

Reply:
Looks like I accidentally went up a major version of `lint-staged`. I got this error:
```
`lint-staged` no longer supports running scripts defined in package.json.

The same behavior can be achieved by changing the command to any of the following:
  - `npm run test-unit -- `
  - `mocha test/unit/ `
```
corresponding with the breaking change here: https://github.com/okonet/lint-staged/releases/tag/v7.0.0

I can go back to the old version if we'd rather not have that.

Reply:
Although it looks like we were already at 7.2.0 actually

Reply:
Okay, that's fine then. Just wanted to clarify. I haven't gotten to that stage of the commit-hook in a while haha